src/components/Policy.js

```diff
@@ -0,0 +1,66 @@
+-To monitor and enforce user compliance with the guidelines of the law and its policies. 
+-To respond to law enforcement requests and as required by applicable law, court order, or governmental regulations.
+</Card.Body>
+                        </Accordion.Collapse>
+                </Card>
+
+                <Card>
+                    <Card.Header>
+                        <Accordion.Toggle as={Button} variant="outline-secondary" eventKey="1">
+                        Sharing Personal Informaion
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="1">
+                            <Card.Body>Crypto Pro will not collect additional categories of personal information or use the personal information we collected for materially different, unrelated, or incompatible purposes without providing you notice. Crypto Pro may disclose your personal information to a third party for a business purpose. When Crypto Pro discloses personal information for a business purpose, Crypto Pro enters a contract that describes the purpose and requires the recipient to both keep that personal information confidential and not use it for any purpose except performing the contract.</Card.Body>
+                        </Accordion.Collapse>
+                </Card>
+
+                <Card>
+                    <Card.Header>
+                        <Accordion.Toggle as={Button} variant="outline-secondary" eventKey="2">
+                        Sales of Personal Information
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="2">
+                            <Card.Body>Crypto Pro does not exchange your personal information for money. Your personal information is secured and is not used to for other unsolicited acts.</Card.Body>
+                        </Accordion.Collapse>
+                </Card>
+
+                <Card>
+                    <Card.Header>
+                        <Accordion.Toggle as={Button} variant="outline-secondary" eventKey="3">
+                        Your Rights and Choices
+                        </Accordion.Toggle>
+                    </Card.Header>
+                        <Accordion.Collapse eventKey="3">
+                            <Card.Body>You have the right to request that Crypto Pro disclose certain information to you about its collection and use of your personal information. After Crypto Pro receives and confirms your verifiable request, it will disclose to you: •	The categories of personal information we collected about you.
+-The categories of sources for the personal information we collected about you.
+-Our business or commercial purpose for collecting that personal information.
+-The categories of third parties with whom we share that personal information.
+-The specific pieces of personal information we collected about you.
+</Card.Body>
```

Comment:
This section is not displaying as a list on the hosted site. Is this expected to be displayed as a list?

Reply:
It doesn't have to be displayed as a list, I'll adjust it and make it into paragraph format